almalinux_build_node.py

```diff
@@ -157,13 +169,25 @@ def sigusr_handler(signum, frame):
 
     node_globals.init_supervisors(config)
     builders = []
+    task_queue = queue.Queue()
```

Comment:
IMO, you need to limit queue size to the amount of workers + 1 or just some fixed size that is configurable and do not request for new tasks when it's full

Reply:
done